packages/mantine/src/components/table/TableLastUpdated.tsx

```diff
@@ -0,0 +1,57 @@
+import {createStyles, DefaultProps, Group, Selectors, Text} from '@mantine/core';
+import {useDidUpdate} from '@mantine/hooks';
+import dayjs from 'dayjs';
+import {FunctionComponent, useState} from 'react';
+import {useTable} from './TableContext';
+
+const useStyles = createStyles((theme) => ({
+    root: {
+        minHeight: '98px',
+    },
+    label: {
+        color: theme.colors.gray[6],
+    },
+}));
+
+type TableLastUpdatedStylesNames = Selectors<typeof useStyles>;
+
+interface TableLastUpdatedProps extends DefaultProps<TableLastUpdatedStylesNames> {
+    /**
+     * Label to contextualize the date
+     *
+     * @default "Last update:"
+     */
+    label?: string;
+    /**
+     * Array of dependencies that triggers an update
+     *
+     * @default []
+     */
+    dependencies?: any[];
```

Comment:
In what use cases do we need to pass this prop?

Reply:
If your table is auto refreshed e.g, with an interval or when you select a facet that is outside the table you need to update the LastUpdated somehow

Reply:
Can we always pass `[data]`?

Reply:
Hmm I was thinking that for a client side table it is not necessary, but it wouldn't hurt. I'm just wondering how we would do it. I'll try something with cloneElement